package.json

```diff
@@ -43,71 +43,71 @@
     "binaryjs": "0.2.1",
     "core-js": "3.6.4",
     "countup.js-angular2": "7.3.0",
-    "date-fns": "^2.16.1",
+    "date-fns": "^2.28.0",
     "debug": "4.1.1",
-    "firebase": "^7.23.0",
-    "firebase-functions": "^3.8.0",
+    "firebase": "^7.24.0",
+    "firebase-functions": "^3.19.0",
     "fuse.js": "^5.2.3",
-    "gsap": "^3.0.5",
+    "gsap": "^3.10.2",
     "hammerjs": "^2.0.8",
     "handlebars": "^4.7.7",
     "iconfonts": "^0.12.0",
     "intersection-observer": "^0.11.0",
     "ionicons": "4.6.3",
     "ios-inner-height": "^1.1.1",
     "lodash": "^4.17.21",
-    "ng-in-viewport": "^6.1.3",
-    "ngx-bootstrap": "^5.6.1",
-    "ngx-cookie-service": "^3.0.4",
+    "ng-in-viewport": "^6.1.5",
+    "ngx-bootstrap": "^5.6.2",
+    "ngx-cookie-service": "^3.1.3",
     "ngx-route-history": "0.0.4",
     "progressbar.js": "1.0.1",
     "rxjs": "6.6.3",
-    "shallow-render": "10.x",
+    "shallow-render": "13.x",
     "time-ago-pipe": "^1.3.2",
-    "ts-key-enum": "^2.0.2",
-    "zone.js": "~0.10.2"
+    "ts-key-enum": "^2.0.11",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "0.1001.4",
-    "@angular-eslint/builder": "1.2.0",
-    "@angular-eslint/eslint-plugin": "1.2.0",
-    "@angular-eslint/eslint-plugin-template": "1.2.0",
-    "@angular-eslint/schematics": "1.2.0",
-    "@angular-eslint/template-parser": "1.2.0",
-    "@angular/cli": "10.1.4",
-    "@angular/compiler-cli": "11.0.0-next.4",
-    "@angular/language-service": "11.0.0-next.4",
+    "@angular-devkit/build-angular": "13.3.1",
```

Comment:
We have a transitive dependency on [sass](https://www.npmjs.com/package/sass) through `@angular-devkit/build-angular`, and upgrading this upgraded sass from `1.26.10` to `1.49.9`. The new version of sass is giving a ton of deprecation warnings during the build process, on the order of thousands of lines: https://github.com/PermanentOrg/web-app/runs/5841449247?check_suite_focus=true#step:6:27 .

I am not familiar with the warnings this specific tool is giving, but in general I find that often deprecated functionality has a non-deprecated alternative that is available in the earlier version, so it might be worth running the automated fixers on the old code in a separate PR before this upgrade.